packages/build-utils/src/edge-function.ts

```diff
@@ -0,0 +1,57 @@
+import type { Files } from './types';
+
+/**
+ * A WASM binding
+ */
+export type EdgeFunctionWasmBinding = {
```

Comment:
Would you prefer a class so we can do `new EdgeFunctionWasmBinding({ name, pathInFiles })`?

Reply:
I don't have a huge preference. For consistency `class` would make sense, but also only seems necessary if we plan on including instance functions.

Reply:
I am going to remove `EdgeFunctionWasmBinding` anyway because it will be inferred from imports.
Builders don't need to think about it--they can import a WASM file and that should just work, which makes the interface much simpler

But `new EdgeFunction` might make sense because all the API is based on classes, so I'll migrate it to that. :)